### ISSUE TITLE
c-lightning decryption key not found

### ISSUE THREAD

**Issue by nettijoe96** [nayutaco/lightning-dissector#28]
I am testing out the new c-lightning beta plugin. I have installed Lua, luarocks, wireshark, and the special lightning-cli. Wireshark can detect lightning packets but cannot decrypt them. 

![wireshark](https://user-images.githubusercontent.com/19435271/52313383-245bae00-2963-11e9-932d-76f592cc6f2e.png)

I am not sure if this is a bug or if I am missing something.

**Comment by nettijoe96** [comment]
Update: 
Although it is still not working, we managed to debug this quite a bit. We noticed that this line is always failing:

secret-factory.lua line 34
`  local key = rex.match(log, length_mac .. " ([0-9a-f]+)")
`
this means that the mac->key pair is not found in the keys.log file.

**Comment by nayuta-gondo** [comment]
You use https://github.com/arowser/lightning?
`Protocols -> LIGHTNING -> Key log file` preference

**Comment by nettijoe96** [comment]
yes this is set in wireshark. I removed the file location for the eclair log and kept it for the c-lightning log. The contents of the keys.log changes on startup of wireshark when I am receiving lightning messages.

**Comment by tock203** [comment]
Thanks for the report! Is that packet sent from c-lightning? or sent from other implementation to c-lightning? Currently lightning-dissector doesn't support the latter.

**Comment by tock203** [comment]
and you have to start Wireshark before c-lightning starts.
